conf/messages

```diff
@@ -8,8 +8,10 @@ sessionTimeout.helpOne = To view your VAT returns, you''ll have to
 sessionTimeout.helpTwo = sign in
 sessionTimeout.helpThree = using your Government Gateway ID.
 
-unauthorised.title = Unauthorised access
-unauthorised.instructions = To view your VAT returns, you''ll have to sign in using your Government Gateway ID.
+unauthorised.title = You''re not authorised to use this service
+unauthorised.youNeedTo = You need to
+unauthorised.signUpWithSoftware = sign up to use software to submit your VAT Returns.
```

Comment:
One of the things that the Standards team in Newcastle said is to not have full stops inside link text.